include/CIndexStoreDB/CIndexStoreDB.h

```diff
@@ -82,7 +82,7 @@ typedef enum {
   INDEXSTOREDB_SYMBOL_ROLE_REL_IBTYPEOF    = 1 << 17,
   INDEXSTOREDB_SYMBOL_ROLE_REL_SPECIALIZATIONOF = 1 << 18,
 
-  INDEXSTOREDB_SYMBOL_ROLE_CANONICAL = 1 << 63,
+  INDEXSTOREDB_SYMBOL_ROLE_CANONICAL = uint64_t(1) << 63,
```

Comment:
This is only valid in C++; you probably want `(uint64_t)1 << 63`

Reply:
Heh, indeed. Thanks.